src/app/send/add-edit.component.ts

```diff
@@ -10,19 +10,18 @@ import { Component } from '@angular/core';
 
 import { SendType } from 'jslib/enums/sendType';
 
-import { ApiService } from 'jslib/abstractions/api.service';
 import { EnvironmentService } from 'jslib/abstractions/environment.service';
 import { I18nService } from 'jslib/abstractions/i18n.service';
 import { PlatformUtilsService } from 'jslib/abstractions/platformUtils.service';
 import { SendService } from 'jslib/abstractions/send.service';
+import { UserService } from 'jslib/abstractions/user.service';
 
 import { SendView } from 'jslib/models/view/sendView';
 import { SendFileView } from 'jslib/models/view/sendFileView';
 import { SendTextView } from 'jslib/models/view/sendTextView';
 
 import { Send } from 'jslib/models/domain/send';
-
-import { SendData } from 'jslib/models/data/sendData';
+import { MessagingService } from 'jslib/abstractions/messaging.service';
```

Comment:
Might want to move this import up with the other abstractions

Reply:
Done